app/controllers/user_items_controller.rb

```diff
@@ -1,13 +1,21 @@
 class UserItemsController < ApplicationController
+  # before_action :set_item, only: %i[new create]
+
   def index
     @user_items = policy_scope(UserItem)
+    @item = Item.new
   end
 
   def create
-    @user_item = UserItem.new(user_item_params)
-    # TODO: Define @user_item.item = item ?
+    @item = Item.new(user_item_params)
+    @item.save!
```

Comment:
In practice, we shouldn't allow the user to create a new item in the Items table. Since your code doesn't work unless they're able to, I can approve this for now but refactor it later once I finish my pg_search PR.